docs/sources/whatsnew/whats-new-next/index.md

```diff
@@ -18,6 +18,60 @@ weight: -37
 
 Welcome to Grafana Cloud! Read on to learn about the newest changes to Grafana Cloud.
 
+## User experience updates in Kubernetes Monitoring
+
+<!-- Beverly Buchanan --> 
+
+_Generally available in Grafana Cloud_
+
+### Quickly troubleshoot with snapshot of issues
+
+Release 1.3.1. To provide quicker identification and troubleshooting, you can view a snapshot of issues on the home page that reach these specific thresholds:
```

Comment:
Are you viewing:
- a snapshot of issues that reach specific thresholds and that view is on the home page OR
- a snapshot of issues that are on the home page + reach specific thresholds?

It's not clear which you mean, so I would recast the sentence to clarify

Reply:
Thanks for the feedback. I'll work on it!